src/v0/destinations/braze/transform.js

```diff
@@ -169,8 +169,8 @@ function getUserAttributesObject(message, mappingJson, destination) {
         if (destKey === 'gender') {
           value = formatGender(value);
         }
-        if (destKey === 'email') {
-          value = value.toLowerCase();
+        if (destKey === 'email' && value !== null) {
+          value = value?.toLowerCase();
```

Comment:
Why optional chaining after null check.
Also shouldn't we be checking for undefined?

Reply:
As per [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), a null value will short-circuit with the value of `undefined`. Hence the check for null. And for email to be undefined, the first check should work.